src/js/pages/system/OverviewDetailTab.js

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import ConfigStore from '../../stores/ConfigStore';
+import DescriptionList from '../../components/DescriptionList';
+import MarathonStore from '../../stores/MarathonStore';
+
+module.exports = class OverviewTab extends mixin(StoreMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      open: false
+    };
+
+    this.store_listeners = [
+      {
+        name: 'config',
+        events: ['ccidSuccess']
+      },
+      {
+        name: 'marathon',
+        events: ['instanceInfoSuccess']
+      }
+    ];
+  }
+
+  componentWillMount() {
```

Comment:
let's do this on `DidMount` because `StoreMixin` begins listening on `DidMount`. if the requests somehow come back insanely fast, we won't hit the listeners